EngineeringModel/ViewModels/ElementDefinitionBrowser/ElementDefinitionTreeRows/ElementUsageRowViewModel.cs

```diff
@@ -385,6 +388,58 @@ protected override void PopulateParameters()
             this.AddParameterBase(addedParameterOrOverride);
         }
 
+        /// <summary>
+        /// Update this <see cref="Tooltip"/> with extra information.
+        /// </summary>
+        protected override void UpdateDetails()
+        {
+            base.UpdateDetails();
+
+            var sb = new StringBuilder(this.Details);
+
+            if(this.ExcludedOptions != null)
+            {
+                sb.AppendLine();
+                if (this.ExcludedOptions.Count == 0)
+                {
+                    sb.AppendLine($"Excluded Options: -");
+                }
+                else
+                {
+                    string excludedOptionsString = $"Excluded Options: ";
+
+                    foreach (Option option in this.ExcludedOptions)
```

Comment:
Please take a look at string.Join().

Reply:
Not the solution I expected:
Instead of the foreach I would do 
sb.AppendLine($"Excluded Options: {string.Join("; ", this.ExcludedOptions.Select(x => x.Name))}");

This is a complete replacement for line 409 - 418.